services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie, Genre, Actor
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__movie__in=genres_ids).distinct()
```

Comment:
The filtering logic for genres is incorrect. Instead of `genres__movie__in=genres_ids`, it should be `genres__id__in=genres_ids` to filter by genre IDs.